static/js/popovers.js

```diff
@@ -347,6 +347,12 @@ exports.register_click_handlers = function () {
         show_message_info_popover(this, rows.id(row));
     });
 
+    $("#main_div").on("click", ".user-mention", function (e) {
+        var row = $(this).closest(".message_row");
```

Comment:
What I discovered so far: 

> row is the object matching the ".message_row" element which is the message having the user mention. It is only used thereafter to parse the message row id, which is passed to `show_message_info_popover()` and it shows the popover of the sender rather than that of the mentioned user as this function is meant to show popover of the sender of the message whose 'zid' is passed.

I think it can tell where it went wrong. (Correct me If I'm wrong)